src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/logic/WorkflowLogic.kt

```diff
@@ -34,18 +46,79 @@ class OrderlyWebWorkflowLogic : WorkflowLogic
         val columnCount = headers.count()
         val paramNames = headers.drop(1)
 
-        return rows.drop(1).mapIndexed { rowIdx, row ->
+        val errors: MutableList<String> = mutableListOf()
+        val reports = rows.drop(1).mapIndexed { rowIdx, row ->
             if (row.count() != columnCount)
```

Comment:
I had to read this a few times because `row.count()` reads so much like "row count", especially when next to a var called `columnCount` and I was confused as to why we'd expect same number of rows as columns :laughing: Seeing as it's also re-used below maybe pull out
 `var numCells = row.count()`? And then also changing `columnCount` to `numCols` would just make it extra clear.